dspace-uclouvain/src/main/java/org/dspace/uclouvain/plugins/UCLouvainAccessStatusHelper.java

```diff
@@ -10,7 +10,7 @@
 import org.dspace.core.Context;
 import org.dspace.services.ConfigurationService;
 import org.dspace.services.factory.DSpaceServicesFactory;
-import org.dspace.uclouvain.factories.UCLouvainResourcePolicyServiceFactory;
+import org.dspace.uclouvain.factories.UCLouvainServiceFactory;
 import org.dspace.uclouvain.services.UCLouvainResourcePolicyService;
```

Comment:
Like in the `CataretroFacultyMetadataConsumer`, no need to import all of `org.dspace.content`